tests/e2e/wp/create.spec.ts

```diff
@@ -196,4 +196,19 @@ test.describe("Creating a new Work Plan", () => {
 
     await wpDashboard.goto();
   });
+
+  test("Admin user can deny a work plan", async () => {
+    const page = adminWpDashboard.page;
+
+    await adminWpDashboard.goto();
+    const unlockButton = page.getByRole("button", { name: "Unlock" }).first();
+    await unlockButton.click();
+    const modal = page.getByRole("dialog");
+    await modal.isVisible();
+    await expect(modal).toContainText("You unlocked this Work Plan");
```

Comment:
Might change this to `await modal.getByRole("heading", { name: "You unlocked this Work Plan" }).isVisible();` to be more specific.